starter-code/index.html

```diff
@@ -1,31 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>Instagram</title>
+ <title>Instagram</title>
+ <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-  Instagram
+ 
+ 
+ 
+ <div class="container">
+ <header>
+   <img src="./images/phones.png" alt="Iphone">
```

Comment:
It's good practice to use semantic html tags, such as <nav>, <header>, <article>, etc.
Here though, the iPhone image isn't exactly a <header>, but more an <aside> I think.
More documentation here: https://www.w3schools.com/html/html5_semantic_elements.asp

Of course this isn't going to impact the mechanics and display of your page, just your code's readability.